src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -58,7 +58,8 @@
   },
   "stsjorbsmod:Brambles": {
     "NAME": "Brambles",
-    "DESCRIPTION": "Deal !D! damage."
+    "DESCRIPTION": "Deal damage equal to the number of stsjorbsmod:Material\u00a0Components played this combat. NL (Deals !D! damage)",
```

Comment:
For the `(Deals !D! damage.)` suffix, take a look at how `Mania` uses `getRawDynamicDescriptionSuffix` and `EXTENDED_DESCRIPTION`; doing it that way avoids having the parenthetical present in the master deck and card selection screens, where it's more confusing than helpful.